app/controllers/application_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationController < ActionController::Base
+  include SessionsHelper
```

Comment:
We should have the [sign in](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_in_method_requirement.png) method and the [sign out ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/session_delete_requirement.png)in the applications controller (along with those for the current user)